cmd/bridge/main.go

```diff
@@ -499,12 +502,36 @@ func main() {
 		Handler: srv.HTTPHandler(),
 	}
 
-	log.Infof("Binding to %s...", httpsrv.Addr)
+	klog.V(2).Infof("Binding to %s...", httpsrv.Addr)
 	if listenURL.Scheme == "https" {
-		log.Info("using TLS")
-		log.Fatal(httpsrv.ListenAndServeTLS(*fTlSCertFile, *fTlSKeyFile))
+		klog.V(2).Info("using TLS")
+		klog.Fatal(httpsrv.ListenAndServeTLS(*fTlSCertFile, *fTlSKeyFile))
 	} else {
-		log.Info("not using TLS")
-		log.Fatal(httpsrv.ListenAndServe())
+		klog.V(2).Info("not using TLS")
+		klog.Fatal(httpsrv.ListenAndServe())
 	}
 }
+
+// initLogs initializes logs the way we want for kubernetes.
+func initLogs() {
+	klog.InitFlags(nil)
+	log.SetOutput(KlogWriter{})
+	log.SetFlags(0)
+}
+
+func setLogLevel(val string) (string, error) {
+	var level klog.Level
+	if err := level.Set(val); err != nil {
+		return "", fmt.Errorf("failed set klog.logging.verbosity %s: %v", val, err)
+	}
+	return fmt.Sprintf("Verbosity set to %s", val), nil
+}
+
+// KlogWriter serves as a bridge between the standard log package and the glog package.
```

Comment:
I don't think you need the bridge, we haven't been using `glog` or `log`, just `capnslog`.